Tests/Tests/PageObjectsTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Threading;
+using NUnit.Framework;
+
+namespace Tests.Tests
+{
+    public class PageObjectsTest : BasicTest
+    {
+        [Test]
+        public void CheckPageObjectsTest()
```

Comment:
The purpose of this test should be revised. Please add a correspondent comment.

Reply:
Test was deleted.